services/port_manager/src/main/resources/application.properties

```diff
@@ -16,6 +16,6 @@ microservices.subnet.service.url=http://localhost:9002
 microservices.route.service.url=http://localhost:9003/routes
 microservices.ip.service.url=http://localhost:9004/ips
 microservices.mac.service.url=http://localhost:9005/macs
-microservices.sg.service.url=http://127.0.0.1:9006/securitygroups
+microservices.sg.service.url=http://localhost:9008
 microservices.node.service.url=http://localhost:9007/nodes
-microservices.dataplane.service.url=http://localhost:9100
\ No newline at end of file
+microservices.dataplane.service.url=http://10.213.43.187:8080/v4/port/
```

Comment:
Please remove local change. We learned the lesson before :-)

Reply:
Yeah ~ I would change it when next submit

Reply:
@kevin-zhonghao Just a reminder.

Reply:
Fix it